FDK/Tools/Programs/autohint/build/osx/xcode4/autohintexe.xcodeproj/project.pbxproj

```diff
@@ -58,6 +43,7 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		56AE0BA71A633867000ECBEE /* libautohintlib.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; name = libautohintlib.a; path = "../../../../../../../../../Library/Developer/Xcode/DerivedData/autohintexe-aqaujhwuqqtoouhccdzyzjbhhmcb/Build/Products/Debug/libautohintlib.a"; sourceTree = "<group>"; };
```

Comment:
@readroberts the path to "libautohintlib.a" referenced here is only temporary and specific to @schriftgestalt Xcode setup at the time he edited the xcodeproj. 
Could that be the reason why I cannot successfully run "BuildAll.sh" fo autohint?
This is clang's error:

    ld: library not found for -lautohintlib  

The only way I can build autohint on OS X is by doing this:
- build `libautohintlib.a` using its own Xcode project `autohintlib.xcodeproj`;
- copy the resulting `autohint/autohintlib/exe/osx/debug/libautohintlib.a` to both `autohint/exe/osx/debug` and to `autohint/exe/osx/release`;
- finally, run autohint's BuildAll.sh script